lib/src/core/better_player.dart

```diff
@@ -36,6 +38,8 @@ class BetterPlayer extends StatefulWidget {
       );
 
   final BetterPlayerController controller;
+  final int? skipDuration;
```

Comment:
duration jest klasa taka, a tutaj dales int i nie widomo co to za jednostka. Proponuje uzyc Duration

Reply:
Ja bym został przy intach bo jesli bede potem porównywał position in millis to ta wartość bedzie w inatach, nazwe ją inaczej i dam do obiektu SkiIntroConfiguration